### ISSUE TITLE
figuring out tabix version

### ISSUE THREAD

**Issue by prasundutta87** [samtools/tabix#32]
Hi,

Is there a way where I can figure out the version of tabix used for indexing a vcf.gz file? 

Regards,
Prasun

**Comment by valeriuo** [comment]
No, `tabix` doesn't burn its version into the output index. How would that information help you?
Also, bear in mind that the latest released and maintained version of `tabix` is part of [HTSlib](https://github.com/samtools/htslib).

**Comment by prasundutta87** [comment]
Thanks @valeriuo. I wanted to provide the version of tabix for a manuscript.

**Comment by valeriuo** [comment]
If you just want the version of `tabix` you are currently using, you can obtain that with the command:
`tabix --version`

**Comment by prasundutta87** [comment]
Actually, an analysis was done approx 1.5 years ago and tabix was the only tool whose version was not recorded. Our cluster system has different versions of tabix (sole installations) and various versions of samtools and bcftools. Tabix was run after running a Picard tool and now it is not known if it was the sole installation one or a part of HTSlib (samtools/bcftools). And hence, this issue.
